### DISCUSSION TITLE
Hot to uninstall?

### DISCUSSION THREAD

**Discussion by GrimpaJ** [Aelto/W3_RandomEncounters_Tweaks#116] [General]
I've tried by deleting every file that is placed in the archive when manually installing.

Result is that the game starts but crashes consistantly during loading screen from main menu.

Is there any documentation on how to delete the mod (as clean as possible)?

**Comment by Aelto** [comment]
Hello, sorry for the delay i decided to finally create the uninstall guide rather than answer you directly. You can find [the article here](https://aelto.github.io/tw3-random-encounters-reworked/indepth-guide/#part-7) where you'll get all the information you need

**Reply by GrimpaJ** [reply]
Thank You very much!
